src/analysis/cauchy_equation.lean

```diff
@@ -0,0 +1,305 @@
+  refine ⟨U, hU0, _⟩,
+  { rw (metric.bounded_iff_subset_ball (0 : ℝ)),
+    use 2 * r,
+    simp only [image_subset_iff],
+    convert subset.trans hUr _,
+    intros x hx,
+    rw mem_sub at hx,
+    rcases hx with ⟨a, b, ha, hb, habx⟩,
+    rw [mem_preimage, mem_ball_iff_norm] at ha,
+    rw [mem_preimage, mem_ball_iff_norm'] at hb,
+    simp only [mem_preimage, mem_closed_ball_zero_iff, ← habx],
+    calc ∥f (a - b)∥ ≤ ∥ f a - f b ∥ : by simp only [map_sub]
+    ... = ∥ (f a - z) + (z - f b) ∥ : by abel
+    ... ≤ ∥ f a - z ∥ + ∥ z - f b ∥  : norm_add_le (f a - z) (z - f b)
+    ... ≤ 2 * r : by linarith }
+end
+
+lemma additive_continuous_at_zero_of_bounded_nbhd_zero (f : ℝ →+ ℝ) {U : set ℝ}
+  (hU : U ∈ nhds (0 : ℝ)) (hbounded : metric.bounded (f '' U)) : continuous_at f 0 :=
+begin
+  rcases (metric.mem_nhds_iff.mp hU) with ⟨δ, hδ, hUε⟩,
+  rcases ((metric.bounded_iff_subset_ball (0 : ℝ)).mp
+    (metric.bounded.mono (image_subset f hUε) hbounded)) with ⟨C, hC⟩,
+  rw continuous_at_iff,
+  intros ε hε,
+  simp only [gt_iff_lt, dist_zero_right, _root_.map_zero, exists_prop],
+  cases (exists_nat_gt (C / ε)) with n hn,
+  obtain hC0 | rfl | hC0 := lt_trichotomy C 0,
+  { simp only [closed_ball_eq_empty.mpr hC0, image_subset_iff, preimage_empty] at hC,
+    rw [subset_empty_iff, ball_eq_empty] at hC,
+    linarith },
+  { simp only [closed_ball_zero] at hC,
+    refine ⟨δ, hδ, λ x hxδ, _⟩,
+    replace hxδ : f x ∈ f '' (ball 0 δ),
+    { simp only [mem_image, mem_ball_zero_iff],
+        refine ⟨x, hxδ, rfl⟩},
+    replace hxδ := mem_of_subset_of_mem hC hxδ,
+    suffices : f x = 0,
+    { simp only [this, norm_zero],
+      exact hε },
+    { simp only [← mem_singleton_iff, hxδ] }},
+  { use δ/n,
+    split,
+    { apply div_pos hδ (lt_trans (div_pos hC0 hε) hn) },
+    { intros x hxδ,
+      have h2 : f (n • x) = n • f x, { exact map_nsmul f x n },
+      have hnpos : 0 < (n : ℝ) := (lt_trans (div_pos hC0 hε) hn),
+      simp only [nsmul_eq_mul] at h2,
+      simp only [mul_comm, ← div_eq_iff (ne.symm (ne_of_lt hnpos))] at h2,
+      rw ← h2,
+      replace hxδ : ∥ x * n ∥ < δ,
+      { simp only [norm_mul, real.norm_coe_nat, ← lt_div_iff hnpos, hxδ], },
+      norm_num,
+      replace hxδ : f (x * n) ∈ f '' (ball 0 δ),
+      { simp only [mem_image, mem_ball_zero_iff],
+        refine ⟨x * n, hxδ, rfl⟩ },
+      rw [div_lt_iff hnpos, ← mem_ball_zero_iff],
+      apply mem_of_subset_of_mem (subset.trans hC _) hxδ,
+      apply closed_ball_subset_ball,
+      rw (div_lt_iff hε) at hn,
+      simpa [mul_comm] using hn }}
+end
+
+lemma additive_continuous_at_zero (f : ℝ →+ ℝ)
+  (h : @measurable ℝ ℝ (real.measurable_space) (borel ℝ) f) : continuous_at f 0 :=
+begin
+  rcases (exists_zero_nbhd_bounded f h) with ⟨U, hU, hbounded⟩,
+  exact additive_continuous_at_zero_of_bounded_nbhd_zero f hU hbounded
+end
+
+lemma continuous_of_measurable (f : ℝ →+ ℝ)
+  (h : @measurable ℝ ℝ (borel ℝ) (borel ℝ) f) : continuous f :=
```

Comment:
```suggestion
  (h : measurable f) : continuous f :=
```
and everywhere else. You can check this is the same by looking at [`real.measurable_space` ](https://leanprover-community.github.io/mathlib_docs/measure_theory/constructions/borel_space.html#real.measurable_space).